packages/product-catalogue/src/components/ListView/index.tsx

```diff
@@ -73,14 +73,6 @@ const ProductCatalogueList = (props: ProductCatalogueListTypes) => {
   }
 
   const pageTitle = `${lang.PRODUCT_CATALOGUE} (${data.length})`;
-  // add a key prop to the array data to be consumed by the table
-  const datasource = data.map((singleObject) => {
-    const prodWithKey = {
-      ...singleObject,
-      key: `${TableColumnsNamespace}-${singleObject.uniqueId}`,
```

Comment:
ref

Reply:
I am not sure how we are getting this uniqueId here or where it drives from .. can you please explain that so that i can see how i can further enhance the implementation.

further more i have added id in the index as well to increase the specificity if id is present 
https://github.com/opensrp/web/pull/742/commits/58d5078acaa48a3fecd5b18226129fa2eeb8c8a9

Reply:
solved by implementing the dataKeyAccessor